index.html

```diff
@@ -1675,11 +1677,12 @@ <h2>The <dfn>NFCPushTarget</dfn> enum</h2>
       </p>
       <p>
         The <dfn>NFCWatchOptions.recordType</dfn> property
-        denotes the string value which is used for matching the
+        denotes the enum value which is used for matching the
         <code>
         <a href="#idl-def-nfcrecordtype">type</a></code> property of each
         <code><a>NFCRecord</a></code> object in a <a>Web NFC message</a>.
-        The default value <code>""</code> means that no matching happens.
+        If the dictionary member is <a>not present</a>, then it will be ignored by the
```

Comment:
Are some changes in the "NFC watch algorithm" algorithm also needed?

Reply:
@foolip Improved dispatch algorithm for handling NFCWatchOptions.recordType, as it was checking for empty string, instead of checking member presence.